src/lib/assets/pods.ts

```diff
@@ -70,6 +70,12 @@ export function watcher(assets: Assets, hash: string) {
         },
         spec: {
           serviceAccountName: name,
+          securityContext: {
+            runAsUser: 1000,
```

Comment:
From what I can tell the image pepr uses currently uses a UID of `65532` https://github.com/chainguard-images/images/tree/main/images/node#compatibility-notes

Reply:
TY i will fix that in the PR

Reply:
TY i will fix that in the PR